### PR TITLE
Fixes #13315: Remove support for old system on rudder server builds

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/13315

### CODE REVIEW

File: rudder-jetty/SPECS/rudder-jetty.spec [hunk 1]
```diff
@@ -34,7 +34,6 @@
 %define ruddervardir     /var/rudder
 %define rudderlogdir     /var/log/rudder
 
-%define jetty_release    9.4.7
 %define date_release     20170914
```
Comment (amousset) [inline rudder-jetty/SPECS/rudder-jetty.spec]: does not seem used either

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -119,24 +104,15 @@ Requires: postgresql >= 8.4
...
-%endif
-
-%if 0%{?rhel} && 0%{?rhel} >= 7
-BuildRequires: java-1.8.0-openjdk-devel selinux-policy-devel
-%endif
-
+## RHEL
 %if 0%{?rhel}
+BuildRequires: java-1.8.0-openjdk-devel selinux-policy-devel
 Requires: mod_ssl jetty-eclipse
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: jetty-eclipse?

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 3]
```diff
@@ -297,17 +273,11 @@ fi
...
-%if 0%{?rhel} && 0%{?rhel} < 7
-chkconfig --add %{apache} 2&> /dev/null
-%endif
-%if 0%{?rhel} && 0%{?rhel} == 6
-chkconfig %{apache} on
-%endif
 # mandatory with systemd wrapper for old init
-%if 0%{?suse_version} && 0%{?suse_version} >= 1315
+%if 0%{?suse_version}
 systemctl daemon-reload
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: you can remove this

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**